### ISSUE TITLE
Error loading colour scheme […]

### ISSUE THREAD

**Issue by hced** [NoahBuscher/Flatron#17]
Hi! I've failed to be able to use this theme. (I'm using Mac OS X 10.9.4 and Sublime 3065).

1. I made a plain vanilla installation of Sublime (by zapping my whole ~/Library/Application\ Support/Sublime Text folder).
2. (Re)Installed Package Control per official instructions.
3. Installed Flatron via Package Control.
4. Added the theme in my user settings … and then things break loose.

![flatron_package_error_msg](https://cloud.githubusercontent.com/assets/93350/4249953/f327ab82-3a7d-11e4-97b3-375e1bd995b1.png)

![flatron_issue](https://cloud.githubusercontent.com/assets/93350/4249959/04aa77fe-3a7e-11e4-8b2f-75254c15cdd5.png)

Console:

```
error: Error loading colour scheme Packages/Theme - Flatron/Flatron/Widget - Flatron.stTheme: Unable to open Packages/Theme - Flatron/Flatron/Widget - Flatron.stTheme
error: Error loading colour scheme Packages/Theme - Flatron/Flatron/Widget - Flatron.stTheme: Unable to open Packages/Theme - Flatron/Flatron/Widget - Flatron.stTheme
error: Error loading colour scheme Packages/Theme - Flatron/Flatron/Widget - Flatron.stTheme: Unable to open Packages/Theme - Flatron/Flatron/Widget - Flatron.stTheme
error: Error loading colour scheme Packages/Theme - Flatron/Flatron/Widget - Flatron.stTheme: Unable to open Packages/Theme - Flatron/Flatron/Widget - Flatron.stTheme
error: Error loading colour scheme Packages/Theme - Flatron/Flatron/Widget - Flatron.stTheme: Unable to open Packages/Theme - Flatron/Flatron/Widget - Flatron.stTheme
error: Error loading colour scheme Packages/Theme - Flatron/Flatron/Widget - Flatron.stTheme: Unable to open Packages/Theme - Flatron/Flatron/Widget - Flatron.stTheme
error: Error loading colour scheme Packages/Theme - Flatron/Flatron/Widget - Flatron.stTheme: Unable to open Packages/Theme - Flatron/Flatron/Widget - Flatron.stTheme
error: Error loading colour scheme Packages/Theme - Flatron/Flatron/Widget - Flatron.stTheme: Unable to open Packages/Theme - Flatron/Flatron/Widget - Flatron.stTheme
error: Error loading colour scheme Packages/Theme - Flatron/Flatron/Widget - Flatron.stTheme: Unable to open Packages/Theme - Flatron/Flatron/Widget - Flatron.stTheme
error: Error loading colour scheme Packages/Theme - Flatron/Flatron/Widget - Flatron.stTheme: Unable to open Packages/Theme - Flatron/Flatron/Widget - Flatron.stTheme
error: Error loading colour scheme Packages/Theme - Flatron/Flatron/Widget - Flatron.stTheme: Unable to open Packages/Theme - Flatron/Flatron/Widget - Flatron.stTheme
error: Error loading colour scheme Packages/Theme - Flatron/Flatron/Widget - Flatron.stTheme: Unable to open Packages/Theme - Flatron/Flatron/Widget - Flatron.stTheme
error: Error loading colour scheme Packages/Theme - Flatron/Flatron/Widget - Flatron.stTheme: Unable to open Packages/Theme - Flatron/Flatron/Widget - Flatron.stTheme
error: Error loading colour scheme Packages/Theme - Flatron/Flatron/Widget - Flatron.stTheme: Unable to open Packages/Theme - Flatron/Flatron/Widget - Flatron.stTheme
error: Error loading colour scheme Packages/Theme - Flatron/Flatron/Widget - Flatron.stTheme: Unable to open Packages/Theme - Flatron/Flatron/Widget - Flatron.stTheme
error: Error loading colour scheme Packages/Theme - Flatron/Flatron/Widget - Flatron.stTheme: Unable to open Packages/Theme - Flatron/Flatron/Widget - Flatron.stTheme
error: Error loading colour scheme Packages/Theme - Flatron/Flatron/Widget - Flatron.stTheme: Unable to open Packages/Theme - Flatron/Flatron/Widget - Flatron.stTheme
error: Error loading colour scheme Packages/Theme - Flatron/Flatron/Widget - Flatron.stTheme: Unable to open Packages/Theme - Flatron/Flatron/Widget - Flatron.stTheme
error: Error loading colour scheme Packages/Theme - Flatron/Flatron/Widget - Flatron.stTheme: Unable to open Packages/Theme - Flatron/Flatron/Widget - Flatron.stTheme
error: Error loading colour scheme Packages/Theme - Flatron/Flatron/Widget - Flatron.stTheme: Unable to open Packages/Theme - Flatron/Flatron/Widget - Flatron.stTheme
error: Error loading colour scheme Packages/Theme - Flatron/Flatron/Widget - Flatron.stTheme: Unable to open Packages/Theme - Flatron/Flatron/Widget - Flatron.stTheme
error: Error loading colour scheme Packages/Theme - Flatron/Flatron/Widget - Flatron.stTheme: Unable to open Packages/Theme - Flatron/Flatron/Widget - Flatron.stTheme
error: Error loading colour scheme Packages/Theme - Flatron/Flatron/Widget - Flatron.stTheme: Unable to open Packages/Theme - Flatron/Flatron/Widget - Flatron.stTheme
error: Error loading colour scheme Packages/Theme - Flatron/Flatron/Widget - Flatron.stTheme: Unable to open Packages/Theme - Flatron/Flatron/Widget - Flatron.stTheme
found 1 files for base name Flatron.sublime-theme
error: Error loading colour scheme Packages/Theme - Flatron/Flatron/Widget - Flatron.stTheme: Unable to open Packages/Theme - Flatron/Flatron/Widget - Flatron.stTheme
error: Error loading colour scheme Packages/Theme - Flatron/Flatron/Widget - Flatron.stTheme: Unable to open Packages/Theme - Flatron/Flatron/Widget - Flatron.stTheme
error: Error loading colour scheme Packages/Theme - Flatron/Flatron/Widget - Flatron.stTheme: Unable to open Packages/Theme - Flatron/Flatron/Widget - Flatron.stTheme
error: Error loading colour scheme Packages/Theme - Flatron/Flatron/Widget - Flatron.stTheme: Unable to open Packages/Theme - Flatron/Flatron/Widget - Flatron.stTheme
error: Error loading colour scheme Packages/Theme - Flatron/Flatron/Widget - Flatron.stTheme: Unable to open Packages/Theme - Flatron/Flatron/Widget - Flatron.stTheme
error: Error loading colour scheme Packages/Theme - Flatron/Flatron/Widget - Flatron.stTheme: Unable to open Packages/Theme - Flatron/Flatron/Widget - Flatron.stTheme
error: Error loading colour scheme Packages/Theme - Flatron/Flatron/Widget - Flatron.stTheme: Unable to open Packages/Theme - Flatron/Flatron/Widget - Flatron.stTheme
error: Error loading colour scheme Packages/Theme - Flatron/Flatron/Widget - Flatron.stTheme: Unable to open Packages/Theme - Flatron/Flatron/Widget - Flatron.stTheme
error: Error loading colour scheme Packages/Theme - Flatron/Flatron/Widget - Flatron.stTheme: Unable to open Packages/Theme - Flatron/Flatron/Widget - Flatron.stTheme
error: Error loading colour scheme Packages/Theme - Flatron/Flatron/Widget - Flatron.stTheme: Unable to open Packages/Theme - Flatron/Flatron/Widget - Flatron.stTheme
error: Error loading colour scheme Packages/Theme - Flatron/Flatron/Widget - Flatron.stTheme: Unable to open Packages/Theme - Flatron/Flatron/Widget - Flatron.stTheme
error: Error loading colour scheme Packages/Theme - Flatron/Flatron/Widget - Flatron.stTheme: Unable to open Packages/Theme - Flatron/Flatron/Widget - Flatron.stTheme
error: Error loading colour scheme Packages/Theme - Flatron/Flatron/Widget - Flatron.stTheme: Unable to open Packages/Theme - Flatron/Flatron/Widget - Flatron.stTheme
error: Error loading colour scheme Packages/Theme - Flatron/Flatron/Widget - Flatron.stTheme: Unable to open Packages/Theme - Flatron/Flatron/Widget - Flatron.stTheme
error: Error loading colour scheme Packages/Theme - Flatron/Flatron/Widget - Flatron.stTheme: Unable to open Packages/Theme - Flatron/Flatron/Widget - Flatron.stTheme
error: Error loading colour scheme Packages/Theme - Flatron/Flatron/Widget - Flatron.stTheme: Unable to open Packages/Theme - Flatron/Flatron/Widget - Flatron.stTheme
error: Error loading colour scheme Packages/Theme - Flatron/Flatron/Widget - Flatron.stTheme: Unable to open Packages/Theme - Flatron/Flatron/Widget - Flatron.stTheme
error: Error loading colour scheme Packages/Theme - Flatron/Flatron/Widget - Flatron.stTheme: Unable to open Packages/Theme - Flatron/Flatron/Widget - Flatron.stTheme
error: Error loading colour scheme Packages/Theme - Flatron/Flatron/Widget - Flatron.stTheme: Unable to open Packages/Theme - Flatron/Flatron/Widget - Flatron.stTheme
error: Error loading colour scheme Packages/Theme - Flatron/Flatron/Widget - Flatron.stTheme: Unable to open Packages/Theme - Flatron/Flatron/Widget - Flatron.stTheme
error: Error loading colour scheme Packages/Theme - Flatron/Flatron/Widget - Flatron.stTheme: Unable to open Packages/Theme - Flatron/Flatron/Widget - Flatron.stTheme
error: Error loading colour scheme Packages/Theme - Flatron/Flatron/Widget - Flatron.stTheme: Unable to open Packages/Theme - Flatron/Flatron/Widget - Flatron.stTheme
error: Error loading colour scheme Packages/Theme - Flatron/Flatron/Widget - Flatron.stTheme: Unable to open Packages/Theme - Flatron/Flatron/Widget - Flatron.stTheme
error: Error loading colour scheme Packages/Theme - Flatron/Flatron/Widget - Flatron.stTheme: Unable to open Packages/Theme - Flatron/Flatron/Widget - Flatron.stTheme
Unable to decode Packages/Theme - Flatron/Flatron/close.png
Unable to decode Packages/Theme - Flatron/Flatron/circle.png
Unable to decode Packages/Theme - Flatron/Flatron/unfold.png
Unable to decode Packages/Theme - Flatron/Flatron/fold.png
Unable to decode Packages/Theme - Flatron/Flatron/folder-closed.png
Unable to decode Packages/Theme - Flatron/Flatron/folder-open.png
Unable to decode Packages/Theme - Flatron/Flatron/btn.png
Unable to decode Packages/Theme - Flatron/Flatron/btn-hover.png
Unable to decode Packages/Theme - Flatron/Flatron/text-field.png
Unable to decode Packages/Theme - Flatron/Flatron/regex.png
Unable to decode Packages/Theme - Flatron/Flatron/casesens.png
Unable to decode Packages/Theme - Flatron/Flatron/wholeword.png
Unable to decode Packages/Theme - Flatron/Flatron/reverse.png
Unable to decode Packages/Theme - Flatron/Flatron/wrap.png
Unable to decode Packages/Theme - Flatron/Flatron/selection.png
Unable to decode Packages/Theme - Flatron/Flatron/lock.png
Unable to decode Packages/Theme - Flatron/Flatron/highlight.png
theme loaded
```

**Comment by NoahBuscher** [comment]
Any luck @rex?

**Comment by rex** [comment]
@NoahBuscher Full disclosure, work has been kicking my ass lately and I just completely forgot about this. I'll give it a whirl sometime today hopefully and post the result!

**Comment by rex** [comment]
Alright @NoahBuscher I just got it installed and working perfectly, and it looks amazing! Steps detailed below.

# tl;dr

**Terminal**
```sh
cd ~/Library/Application\ Support/Sublime\ Text\ 3/Packages
git clone git@github.com:NoahBuscher/Flatron.git Theme\ -\ Flatron
```

**Sublime Preferences**
```json
{
    "theme": "Flatron.sublime-theme",
    "color_scheme": "Packages/Theme - Flatron/Flatron.tmTheme"
}
```

# Annotated steps

1) Terminal
---
`cd ~/Library/Application\ Support/Sublime\ Text\ 3/Packages`
`git clone git@github.com:NoahBuscher/Flatron.git Theme\ -\ Flatron`

![screenshot 2015-01-21 12 09 44](https://cloud.githubusercontent.com/assets/4247754/5842034/3e1f2228-a167-11e4-9636-95df9c517916.png)

2) Sublime preferences - Set theme
---
```json
{
    "theme": "Flatron.sublime-theme"
}
```
![screenshot 2015-01-21 12 11 36](https://cloud.githubusercontent.com/assets/4247754/5842065/90b93a82-a167-11e4-92e1-a7900f64565c.png)

3) Sublime preferences - Set color scheme
---
```json
{
    "color_scheme": "Packages/Theme - Flatron/Flatron.tmTheme"
}
```
![screenshot 2015-01-21 12 24 34](https://cloud.githubusercontent.com/assets/4247754/5842185/8397c91c-a168-11e4-9462-5a27604188c8.png)

Looks amazing! :smile: :beers: 

I **do** wish there was a way to tighten up the directory listing, but that lies outside the scope of this issue.

**Comment by peterkos** [comment]
@rex Just edit the "row_padding" array in the `Flatron.sublime-theme ` file, on approximately [line 445](https://github.com/NoahBuscher/Flatron/blob/master/Flatron.sublime-theme#L445).

**Comment by rex** [comment]
@peterkos Thank you for pointing me to that!!! I adjusted that line to read 

```json
{
    "row_padding": [2,2]
}
```

And the result is a much tighter but still-organized sidebar tree (this is just my own personal preference, of course!)

![screenshot 2015-01-21 13 08 29](https://cloud.githubusercontent.com/assets/4247754/5843030/22cec098-a16f-11e4-9f84-57eb23116fef.png)

I am excited now because I can go into the `sublime-theme` file and make changes without fear of losing them to an update. 

Thank you!!

**Comment by rex** [comment]
Alright I lied. One more. This issue inspired me to write a simple bash installer for users on Macs. No dependencies, of course, and it should be able to run on any Mac with a valid Sublime Text 3 installation without installing any other software.

**Comment by NoahBuscher** [comment]
Thank you all so much for your help! I have finally found out what the issue was regarding the Package Manager and now you can install with just a few keyboard taps!

The manual install script is now in the repo, and is available for anyone to use.

Again, thank you so much!

**Comment by rex** [comment]
That's exciting news :D 

Just curious, does Sublime auto-resolve theme asset includes to a `Theme - $theme_name` format? For example, `Flatron/foo/bar.png` would resolve to `Theme - Flatron/foo/bar.png`?

**Comment by NoahBuscher** [comment]
I *thought* so previously, but it appears not. That's why the package manager was not working, so I had to change the paths to just `/Flatron`.

**Comment by rex** [comment]
Ah, I see. Well at least it was a simple fix! I'm glad that Flatron finally got the TLC it needed :) :beers:
